python/lib/dcoscli/tests/data/calico/flag_parsing.txt

```diff
@@ -0,0 +1,2 @@
+Failed to execute command: open calico-default.yaml: no such file or directory
```

Comment:
Current messgae `Error: unknown shorthand flag: 'f' in -f` from Cobra not from CalicoCtl